_includes/tutorials/produce-consume-lang/scala/markup/test/2_1-write-test-trait.adoc

```diff
@@ -0,0 +1,10 @@
+First we create a `trait` that will extend `AnyFlatSpec`, `Matchers` and other tests traits.
+Because theses tests might be asynchronous we will use the scalatest functions from `Eventually`.
+And because all Kafka Clients related tests will extends this trait let's call it:
+`src/test/scala/io/confluent/developer/KafkaFlatSpec.scala`
```

Comment:
```suggestion
Because theses tests might be asynchronous, we will use the scalatest functions from `Eventually`. 

All Kafka Clients related tests will extends this trait, so let's call it:
`src/test/scala/io/confluent/developer/KafkaFlatSpec.scala`
```